torch/fft/__init__.py

```diff
@@ -549,3 +555,177 @@
     tensor([ 2.0000+-0.0000j, -0.5000-0.6882j, -0.5000-0.1625j, -0.5000+0.1625j,
         -0.5000+0.6882j])
 """)
+
+fftfreq = _add_docstr(_fft.fft_fftfreq, r"""
+fftfreq(n, d=1.0, *, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+
+Returns Discrete Fourier Transform sample frequencies for signal size :attr:`n`.
+
+Note:
+
+    By convention, the FFT returns positive frequency terms first, followed by
+    the negative frequencies in reverse order, so that ``f[-i]`` in python
+    gives the negative frequency terms. For an FFT of length :attr:`n` and
+    with inputs spaced in length unit :attr:`d`, the frequencies are::
+
+        f = [0, 1, ..., (n - 1) // 2, -(n // 2), ..., -1] / (d * n)
+
+Note:
+    For even lengths, the Nyquist frequency at ``f[n/2]`` can be thought of as
+    either negative or positive. :func:`~torch.fft.fftfreq` follows NumPy's
+    convention of taking it to be negative.
+
+
+Args:
+    n (int): the FFT length
+    d (float, optional): The sampling length scale.
+        The unit length (or time) between individual samples in the FFT input.
+        If given, the frequencies are given in physical units of 1/unit.
```

Comment:
I don't think I follow what this last sentence is getting. Per the above formula, `d` just divides all the frequencies? 

Maybe there's a way to briefly explain what this does (divide the frequency samples) followed by another sentence that explains its conceptual perspective. Maybe something like: "Conceptually, the time between samples in the FFT input"?